### PR TITLE
Doc/issue139 man pages config statusbar config

### PR DESCRIPTION
**Pull request by gungula**
Re-worked the man pages.
oxide-config and oxide-bar: fixed errors and typos
oxide and oxide-msg: some small formatting things

Changed the config.yml: Using meta instead of alt now.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,8 +5,10 @@ TARGET_DIR := /usr/bin
 CONFIG_DIR := /etc
 
 define gen_manpages
-	pandoc --standalone --to man $(ROOT_DIR)/man/src/oxide-msg.1.md -o $(ROOT_DIR)/man/oxide-msg.1
 	pandoc --standalone --to man $(ROOT_DIR)/man/src/oxide.1.md -o $(ROOT_DIR)/man/oxide.1
+	pandoc --standalone --to man $(ROOT_DIR)/man/src/oxide-bar.1.md -o $(ROOT_DIR)/man/oxide-bar.1	
+	pandoc --standalone --to man $(ROOT_DIR)/man/src/oxide-config.1.md -o $(ROOT_DIR)/man/oxide-config.1
+	pandoc --standalone --to man $(ROOT_DIR)/man/src/oxide-msg.1.md -o $(ROOT_DIR)/man/oxide-msg.1
```
Comment (janschaible) [inline Makefile:11]: You could just do this for all files in the folder ?:)
so a new file would not need to be added in here ?:)

File: Makefile [hunk 2]
```diff
@@ -26,25 +28,38 @@ install:
...
 	sudo install -Dm644 $(ROOT_DIR)/resources/oxide.desktop $(SHARE_DIR)/xsessions/oxide.desktop
-	sudo cp $(ROOT_DIR)/man/oxide-msg.1 $(ROOT_DIR)/man/oxide.1 $(SHARE_DIR)/man/man1/
+
+	$(call gen_manpages)
+	sudo cp $(ROOT_DIR)/man/oxide.1 \
+		$(ROOT_DIR)/man/oxide-bar.1 \
+		$(ROOT_DIR)/man/oxide-config.1 \
+		$(ROOT_DIR)/man/oxide-msg.1 \
+		$(SHARE_DIR)/man/man1/
+
```
Comment (janschaible) [inline Makefile:45]: If you create a folder where you generate the manpages to, you could simply copy the folder ?:)

File: man/src/oxide-config.1.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+## WORKSPACE_ARGS
+
+Next
+: next initialized workspace with a higher index than the current workspace
+
+Previous
+: next initialized workspace with a lower index than the current workspace
+
+Next_free
+: next available workspace with a higher index than the current workspace which is not initialized
```
Comment (janschaible) [inline man/src/oxide-config.1.md:117]: Next
: next initialized workspace with a higher index than the current workspace. If the workspace with the highest index is selected, the index with the lowest index will be selected.

Previous
: next initialized workspace with a lower index than the current workspace. If the workspace with the lowest index is selected, the index with the highest index will be selected.

Next_free
: next available workspace with which is not initialized. Gaps in the workspace indices are filled first.

?:)

**Review by janschaible — APPROVED** [review]
@gungula nice looking pr ;)

File: man/src/oxide-config.1.md (hunk 3 above)
Comment (gungula) [reply man/src/oxide-config.1.md:117]: Good idea, thank you 👍🏼

File: Makefile (hunk 1 above)
Comment (gungula) [reply Makefile:11]: @FelixSchladt ?

File: Makefile (hunk 2 above)
Comment (gungula) [reply Makefile:45]: @FelixSchladt ?

**Comment by gungula** [comment]
@FelixSchladt 

The section WORKSPACE_ARGS is a little different in `oxide-msg` and `oxide-config`. In my understanding the arguments should be the same, shouldn't they? In `oxide-msg` there are `next`, `previous`, `new` and `index` while in `oxide-config` there is no `new` but `next_free`.

What is your opinion to this?

image

File: Makefile (hunk 1 above)
Comment (FelixSchladt) [reply Makefile:11]: i tried but i dont think pandoc offers this functionality...

I could write a small shell script but in the end manual adding of files is required anyway in the install/uninstall part and I dont think there is gonna be many more manpages in the future so i see no value

**Comment by FelixSchladt** [comment]
> @FelixSchladt
> 
> The section WORKSPACE_ARGS is a little different in `oxide-msg` and `oxide-config`. In my understanding the arguments should be the same, shouldn't they? In `oxide-msg` there are `next`, `previous`, `new` and `index` while in `oxide-config` there is no `new` but `next_free`.
> 
> What is your opinion to this?
> 
> image

There were some changes...

So the next free is the newer one. Please adjust the oxide-msg man page accordingly

File: Makefile (hunk 2 above)
Comment (FelixSchladt) [reply Makefile:45]: still manual adding is required in the uninstall phase, i would just leave it as is

**Review by FelixSchladt — APPROVED** [review]

**Pull request merged**